server/aws-lsp-yaml-json/src/language-service/jsonLanguageService.ts

```diff
@@ -21,7 +21,7 @@ export class JsonLanguageService implements AwsLanguageService {
             schemaRequestService: props.uriResolver?.bind(this),
         })
 
-        const schemas = props.defaultSchemaUri ? [{ fileMatch: ['*.json'], uri: props.defaultSchemaUri }] : undefined
+        const schemas = props.defaultSchemaUri ? [{ fileMatch: ['*'], uri: props.defaultSchemaUri }] : undefined
```

Comment:
Have you intentionally changed the extension in `fileMatch` field? What's the idea for any extension, should client define for which file extensions this server will be used?

Reply:
discussed this comment in slack, this change is ok

Reply:
@aderende Could you also share the reasoning here, please?

Reply:
I replaced `fileMatch: ['*.yaml']` with `fileMatch: ['*']` every file’s URI in internal destination new API is "*" and it doesn't match "*.yaml" pattern.